sphinx_book_theme/__init__.py

```diff
@@ -22,6 +29,28 @@ def add_static_path(app):
     app.config.html_static_path.append(str(static_path))
 
 
+def find_url_relative_to_root(pagename, relative_page, path_docs_source):
```

Comment:
this is because when the sidebar is rendered, we only have the path to each item *relative to the current page*. whereas when we store page names in the toctree directive, they are always relative to the *site root*.